muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/chat/ChatRoomSyncScheduler.java

```diff
@@ -43,21 +45,25 @@
 
   @Inject
   private WorkspaceController workspaceController;
+  
+  @Inject
+  private UserChatSettingsDAO userChatSettingsDao;
 
   @Schedule(second = "0", minute = "*/15", hour = "*", persistent = false)
   @TransactionAttribute(TransactionAttributeType.REQUIRED)
   public void updateChatRooms() {
-    String enabledUsersCsv = pluginSettingsController.getPluginSetting("chat", "enabledUsers");
-    if (enabledUsersCsv == null) {
-      return;
-    }
-    List<String> enabledUsers = Arrays.asList(enabledUsersCsv.split(","));
 
     String enabledWorkspacesCsv = pluginSettingsController.getPluginSetting("chat", "enabledWorkspaces");
     if (enabledWorkspacesCsv == null) {
-      return;
+     return;
     }
     List<String> enabledWorkspaces = Arrays.asList(enabledWorkspacesCsv.split(","));
+	  
+	List<UserChatSettings> listUsers = userChatSettingsDao.listAll();
```

Comment:
`DAO.listAll()` lists all rows from the table. What is the expectation for the row count, does every user have an UserChatSettings, or does just a few users have a row? The synchronization can get too time consuming if there are many rows and in that case the operation needs to be split into chunks. That can be added as another issue though.

Reply:
Was this issue added?